index.html

```diff
@@ -1194,16 +1197,104 @@
             This section is currently work in progress, and subject to change.
           </div>
           <p>
+            There are several use-cases which are a good fit to be, at least
+            partially, implemented on the GPU, such as motion recognition,
+            pattern recognition, background removal, as well as 3D point cloud.
+          </p>
+          <p>
+            This section explains which APIs can be used for some of these
+            mentioned use-cases; the concrete usage examples are provided in
+            the <a href=
+            "https://www.w3.org/wiki/Media_Capture_Depth_Stream_Extension#Examples">
+            Examples</a> section.
```

Comment:
copying approach from other places, prefer not to have bellow together with link.